pkg/virt-controller/watch/vm.go

```diff
@@ -1285,6 +1294,38 @@ func (c *VMController) updateStatus(vmOrig *virtv1.VirtualMachine, vmi *virtv1.V
 	return err
 }
 
+func (c *VMController) copyReadyConditionFromVMI(vm *virtv1.VirtualMachine, vmi *virtv1.VirtualMachineInstance) {
```

Comment:
nit: rather call it `syncReadyConditionFromVMI`?

Reply:
Done